### ISSUE TITLE
Fix false positives for :is selector in selector-pseudo-class-no-unknown

### ISSUE THREAD

**Issue by silverwind** [stylelint/stylelint#4236]
> Clearly describe the bug

[`:is`](https://developer.mozilla.org/en-US/docs/Web/CSS/:is) selector is not known by this rule.

> Which rule, if any, is the bug related to?

`selector-pseudo-class-no-unknown`

> What CSS is needed to reproduce the bug?

e.g.

```css
*:is(*) { color: red; }
```

> What stylelint configuration is needed to reproduce the bug?

e.g.

```json
{
  "rules": {
    "selector-pseudo-class-no-unknown": "always"
  }
}
```

> Which version of stylelint are you using?

`10.1.0`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

CLI

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No

> What did you expect to happen?

`:is` to not trigger the rule

> What actually happened (e.g. what warnings or errors did you get)?

`:is` triggering the rule

```shell
 9011:3   ✖  Unexpected unknown pseudo-class selector ":is"   selector-pseudo-class-no-unknown
```

**Comment by hudochenkov** [comment]
Thanks for the report and for using the template.

Consider contributing by updating this set https://github.com/stylelint/stylelint/blob/75326e9183e60be8141d4bef6f9dcfaeb2a2b777/lib/reference/keywordSets.js#L247

**Comment by silverwind** [comment]
Will send a PR later.
